app/src/main/res/layout/walkthrough_final_fragment.xml

```diff
@@ -12,5 +16,121 @@
   <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
This should be inside scrollview.
![Screenshot_1585842615](https://user-images.githubusercontent.com/9396084/78270365-5bf2c580-7528-11ea-94ec-6fd913ccec7e.png)

Reply:
updated